bin/bdr_check

```diff
@@ -2,4 +2,27 @@
 
 require "json"
 
-puts [].to_json
+require 'bosh_deployment_resource'
+
+request = JSON.parse(STDIN.read)
+
+source = request.fetch("source")
+target = source["target"] || ""
+
+auth = BoshDeploymentResource::Auth.parse(source)
+ca_cert = BoshDeploymentResource::CaCert.new(source["ca_cert"])
```

Comment:
This probably needs to use the same changes made in `bdr_out` for the cert yeah?